rotkehlchen/tests/api/test_nfts.py

```diff
@@ -148,41 +153,261 @@ def test_nft_ids_are_unique(rotkehlchen_api_server):
     assert len(all_ids) == len(set_of_ids)
 
 
-@requires_env([TestEnvironment.NIGHTLY, TestEnvironment.NFTS])
-@flaky(max_runs=3, min_passes=1)  # all opensea calls have become quite flaky
-@pytest.mark.parametrize('ethereum_accounts', [[TEST_ACC4]])
+# @requires_env([TestEnvironment.NIGHTLY, TestEnvironment.NFTS])
+# @flaky(max_runs=3, min_passes=1)    # all opensea calls have become quite flaky
```

Comment:
What are you doing in those 2 lines? Why did you comment those?

Reply:
Ah oh, my bad. Commented them out when was testing locally, will undo it